test/integration/targets/get_certificate/tests/validate.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Test get_certificate
+  connection: local
+  hosts: localhost
+  gather_facts: false
+  tasks:
+
+    - name: Get red hats certificate
+      get_certificate:
+        host: www.redhat.com
+        port: 443
+      register: result
+
+    - debug: var=result
```

Comment:
did you mean to leave this `debug` in?

Reply:
I left it in so that if a test fails you can see the cert the last step got (i.e. like the cert on RH.com changes). It will leave extra lines in the output but you won't have to push again if something is failing. Do you want it removed?